fdbmonitor/fdbmonitor.cpp

```diff
@@ -642,6 +693,31 @@ CSimpleOpt::SOption g_rgOptions[] = { { OPT_CONFFILE, "--conffile", SO_REQ_SEP }
 	                                  { OPT_HELP, "--help", SO_NONE },
 	                                  SO_END_OF_OPTIONS };
 
+// Return resident memory in bytes for the given process, or 0 if error.
+uint64_t getRss(ProcessID id) {
+#ifndef __linux__
+	// TODO: implement for non-linux
```

Comment:
Can we please print a warning here?

Reply:
Added a warning when the option is being parsed. Btw for win it doesn't currently support `--memory`. For other systems the `--memory` will now still pass to fdbserver as command line argument, where the argument is supported.